app/Http/Requests/Admin/Settings/UpdateSettingsRequest.php

```diff
@@ -34,17 +36,30 @@ public function rules() {
 			$rules[$field] = 'required|string';
 		};
 
-		$rules['invoices_accountant'] = 'required|email';
+		$rules['accountant_accountant_email'] = 'required|email';
```

Comment:
feels like you could just call them "accountant_email" and "accountant_password"